Controller/ProfileManager.php

```diff
@@ -36,29 +37,28 @@ class ProfileManager
      * @param FormFactory $formFactory
      */
     public function __construct(
-        UserDiscriminator $userDiscriminator,
-        ContainerInterface $container,
-        ProfileController $controller,
-        FormFactory $formFactory
-    ) {
+    UserDiscriminator $userDiscriminator, ContainerInterface $container,
```

Comment:
Restored.
      From: Massimiliano Arione <notifications@github.com>
 To: PUGX/PUGXMultiUserBundle <PUGXMultiUserBundle@noreply.github.com> 
Cc: George Brooks <truckeetrout@yahoo.com> 
 Sent: Thursday, November 5, 2015 12:10 AM
 Subject: Re: [PUGXMultiUserBundle] Add form options, controller template arrays (#95)
   
In Controller/ProfileManager.php:> @@ -36,29 +37,28 @@ class ProfileManager
>       * @param FormFactory $formFactory
>       */
>      public function __construct(
> -        UserDiscriminator $userDiscriminator,
> -        ContainerInterface $container,
> -        ProfileController $controller,
> -        FormFactory $formFactory
> -    ) {
> +    UserDiscriminator $userDiscriminator, ContainerInterface $container,
This change is useless and against PSR-2. Please restore previous line splitting.—
Reply to this email directly or view it on GitHub.